### PR TITLE
misc. cleanups

### PR DESCRIPTION
**Pull request by pq**
* cleans up some unused imports
* does some lambda migrations
* adds missing annotations

Not complete; just opportunistic.

/cc @stevemessick @jacob314 @kenzieschmoll

### CODE REVIEW

File: src/io/flutter/server/vmService/DartVmServiceDebugProcess.java [hunk 1]
```diff
@@ -24,60 +24,44 @@
 import com.intellij.testFramework.LightVirtualFile;
 import com.intellij.util.BitUtil;
 import com.intellij.util.TimeoutUtil;
-import com.intellij.xdebugger.XDebugProcess;
-import com.intellij.xdebugger.XDebugSession;
-import com.intellij.xdebugger.XDebugSessionListener;
-import com.intellij.xdebugger.XDebuggerBundle;
-import com.intellij.xdebugger.XSourcePosition;
+import com.intellij.xdebugger.*;
```
Comment (stevemessick) [inline src/io/flutter/server/vmService/DartVmServiceDebugProcess.java]: This one is just going to keep changing, depending on who formats it. The AS rules use the expanded form.

Comment (pq) [reply src/io/flutter/server/vmService/DartVmServiceDebugProcess.java]: Ah, sorry.  The actual setting is a little further down:

image

Comment (pq) [reply src/io/flutter/server/vmService/DartVmServiceDebugProcess.java]: Thinking on this some more, I suggest I just land.  Until we all get unified behind a shared setting this churn will continue.  Once we unify, we could consider a sweeping PR to canonicalize...

Comment (pq) [reply src/io/flutter/server/vmService/DartVmServiceDebugProcess.java]: https://github.com/flutter/flutter-intellij/issues/2922 to follow-up.

**Comment by stevemessick** [comment]
It's already approved for merging :)

**Pull request merged**